doc/schemas/mccf/2023-06-01-preview/examples/Ballots_Get.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We want to render `mccfgov.json` with `sphinxcontrib.openapi`. When it parses that OpenAPI, it blows up if the referenced example files aren't present. So we copy the examples as well, from the [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/confidentialledger/data-plane/Microsoft.ManagedCcf/preview/2023-06-01-preview) repo. However, since the OpenAPI is v2 rather than v3, the sphinx plugin can't actually _use_ the examples, and throws another error if we modify the directive to try and render them.

I'll see if we can remedy that, but including the files for now seems like the quick fix.

Reply:
That's unfortunate, including them does seem like the best solution for now. It may be worth checking the same of typespec docgen tooling, which is hopefully coming along, and which we can hopefully leverage at some point.

Reply:
It looks like typespec may be supported [through typedoc](https://microsoft.github.io/typespec/language-basics/documentation#tsdoc-doc-comments), which we already use and have (some) cross-linking support for with Sphinx.

Reply:
I'll take a look at this, but don't have high hopes. We really want to document the HTTP API, to a level where users can use it from their own clients. The TypeSpec source definitions are likely to be an unhelpful level of abstraction, since they talk about TypeSpec types (and operation names), rather than HTTP paths, requests, headers, and bodies. Perhaps we can determine "the" HTTP representation for these with the TypeDoc tooling, but I believe the best way to do that is via OpenAPI.

Another approach, where we _may_ end up, is generating clients from TypeSpec - in that case I hope the (generated) docs for those (generated) clients can also be picked up by Sphinx, and included in the core CCF docs. But I